### PR TITLE
Update to lookup automox host from slack

### PR DESCRIPTION
**Pull request by nhurd-r7**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [x] Workflow verified as working in a test
- [x] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [x] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [x] Workflow includes a detailed Description
- [x] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [x] Triggers, Decisions, and other workflow steps include descriptions where needed
- [x] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [x] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [x] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [x] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

```
### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

<img width="491" alt="slack_lookup_host" src="https://user-images.githubusercontent.com/74786214/155595168-6ce934ad-cfbd-4aeb-9745-6421d13602e5.png">
<img width="375" alt="slack_thread_success" src="https://user-images.githubusercontent.com/74786214/155595169-c9357858-75f5-49e4-bd01-0a718462416b.png">
![workflow](https://user-images.githubusercontent.com/74786214/155595171-946050e2-9600-4241-81f4-5a6f10bd5e71.png)

### CODE REVIEW

**Comment by nhurd-r7** [comment]
The original workflow used the 'lookup-host' trigger syntax, which is already used by an IVM workflow available in the library. This causes collisions as ICON is not able to distinguish which workflow should be run when they use the same trigger syntax. To avoid this error, I've updated the workflow to trigger from 'lookup-automox-host', and updated the screenshots and documentation accordingly.

**Comment by llaszuk-r7** [comment]
@nhurd-r7 Hey, workflow validators failing, try to fix it
```
Validator "WorkflowChangelogValidator" failed! 
	Cause: Version history of help.md missing version 2.0.0. Please add missing version.

Validator "WorkflowTitleValidator" failed! 
	Cause: Title should not start with a lower case letter in .icon file.
```

File: workflows/Lookup_Automox_Host_From_Slack/workflow.spec.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ products: ["insightconnect"]
 name: Lookup_Automox_Host_From_Slack
 title: "Lookup Automox Host from Slack"
 description: Lookup an Automox managed host by hostname or IP address from Slack to quickly collect details related to host details, policy compliance and status, and upcoming patch details.
-version: 1.0.0
+version: 2.0.0
```
Comment (llaszuk-r7) [inline workflows/Lookup_Automox_Host_From_Slack/workflow.spec.yaml:6]: Please add version update in help.md change log

Comment (nhurd-r7) [reply workflows/Lookup_Automox_Host_From_Slack/workflow.spec.yaml:6]: Done!

**Review by llaszuk-r7 — APPROVED** [review]

**Review by ablakley-r7 — APPROVED** [review]

**Review by mberezin-r7 — APPROVED** [review]
> Kindly do not change the folder name from capital to small, as there is an outstanding issue: https://issues.corp.rapid7.com/browse/SOAR-8751

I approved. Besides the folder renaming issue mentioned above, the `lookup-automox-host` changes in this workflow look good to me.
We can merge this PR as is once the above ticket is resolved, or if we undo the folder renaming I can re-approve this PR. My vote would be to wait until the bug/ticket is resolved.

**Comment by mcochran-r7** [comment]
closed due to age

**Pull request closed without merging**